projects/components/src/table/table.component.ts

```diff
@@ -561,14 +579,52 @@ export class TableComponent
     };
   }
 
-  @HostListener('mousemove', ['$event'])
   public onResizeMouseMove(event: MouseEvent): void {
-    this.debouncedResizeColumn(event);
+    if (this.resizable && this.resizeStartX > 0 && !isNil(this.columnResizeHandler)) {
+      this.columnResizeHandler.style.right = `${this.resizeStartX - event.clientX}px`;
+    }
+  }
+
+  public onResizeMouseUp(event: MouseEvent): void {
+    this.checkAndResizeColumn(event);
+    this.changeDetector.detectChanges();
+  }
+
+  private addEventListeners(): void {
+    this.document.addEventListener('mousemove', event => this.onResizeMouseMove(event));
```

Comment:
We could attach it on the root element (inject element). We could also use rxjs but not a requirement